examples/matchmaker/mo/mo.go

```diff
@@ -0,0 +1,222 @@
+package mo
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/GoogleCloudPlatform/open-match/examples/matchmaker/divs"
+	"github.com/GoogleCloudPlatform/open-match/internal/pb"
+	"github.com/gobs/pretty"
+	"github.com/gogo/protobuf/jsonpb"
+)
+
+var (
+	teamSize int
+	numTeams int
+	debug    bool
+	regions  []string
+	tiers    []string
+)
+
+type poolArray []*pb.PlayerPool
+type ftrArray []*pb.Filter
+type rosterArray []*pb.Roster
+
+/*
+//example of how to use this module
+func main() {
```

Comment:
Consider converting this to a golang example test file. 
https://blog.golang.org/examples

Reply:
deleted